geometry_msgs/Pose[] candidate_views
---
bool success